octoprint_octobullet/__init__.py

```diff
@@ -29,10 +33,106 @@ def __init__(self):
 		self._bullet = None
 		self._channel = None
 		self._sender = None
+		self._pc_interval = 5
+		self._message_count = 0
+		self._quiet_time_sec = 1800
+		self._last_message = 0
+		self._etl_format   = "{days:d}:{hours:02d}h {minutes:02d}m"
```

Comment:
That's a weird format I think: "0:23h 12m"? Either it should be "0:23:12" (that would be easy to mistake for hours:minutes:seconds though) or "0d 23h 12m", or even better "0d 23h 12min" (because "m" is actually "meters" and not "minutes").